api/v1/firewall_types.go

```diff
@@ -69,6 +69,8 @@ type Data struct {
 	Interval string `json:"interval,omitempty"`
 	// DryRun if set to true, firewall rules are not applied
 	DryRun bool `json:"dryrun,omitempty"`
+	// LogAcceptedConnections if set to true, also log accepted connections in the droptailer log
+	LogAcceptedConnections bool `json:"logAcceptedConnections,omitempty"`
```

Comment:
I am not sure if it can break existing firewall-controllers due to the signature when we put this in here? Have you tried? 

I thought that altering the content of this struct will change the JSON on which the signature is calculated and then there will be a mismatch between the signature calculated by the GEPM and the older firewall-controller versions.

If this is the case, let's move it to the firewall spec directly and think about how to resolve #104.